tests/golden/check-anchors/expected.gold.license

```diff
@@ -0,0 +1,2 @@
+# SPDX-FileCopyrightText: 2022 Serokell <https://serokell.io>
```

Comment:
It would probably be more convenient to add `tests/golden/**/*.gold` to `.reuse/dep5` to avoid having to create these standalone license files :thinking:  Can you please do that? And then delete this file and the one in the `check-localhost` directory.